_posts/2019-07-10-cockpit-198.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Cockpit 198
+author: pitti
+date: 2019-07-10
+tags: cockpit linux
+slug: cockpit-198
+category: release
+summary: Cockpit with PatternFly 4 user interface design update
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 198.
+
+### PatternFly4 user interface design
+
+Cockpit got restyled to match the
+[PatternFly 4](https://pf4.patternfly.org/components/Button/examples/)
+User Interface design, including the
+[Red Hat Text and Display fonts](https://github.com/RedHatOfficial/RedHatFont).
+
+This aligns Cockpit with other web user interfaces that use
```

Comment:
```suggestion
This style refresh aligns Cockpit with other web user interfaces that use
```

It's almost always a good idea to say what "this" is, especially when it's in a different paragraph.